File: app-ui/MediWeckerUI/Application/Features/Notifications/RescheduleSuggestion.cs
namespace MediWeckerUI.Application.Features.Notifications;

public class RescheduleSuggestion
{
    public long MedicineId { get; set; }
    public RescheduleSuggestionType Type { get; set; }
    public long SuggestedTimeFromMidnight { get; set; }
}

File: app/src/main/java/de/htwBerlin/ai/mediAlarm/reminderPrompt/RescheduleSuggestion.kt
package de.htwBerlin.ai.mediAlarm.reminderPrompt

import java.util.UUID

data class RescheduleSuggestion(
    val medicineId: Long,
    val alarmId: Long,
    val type: RescheduleSuggestionType,
    val suggestedTimeFromMidnight: Long
)
